.github/workflows/deploy-scripts-internally.yaml

```diff
@@ -0,0 +1,43 @@
+name: Deploy Scripts Internally
+
+on:
+    push:
+        branches:
+            - master
+
+concurrency:
+    group: templates-${{ github.event.pull_request.number || github.ref }}
+    cancel-in-progress: true
+
+jobs:
+    template-upload:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v4
+
+            - name: Install nango CLI
+              run: |
+                  npm install -g nango
+
+            - name: Process and deploy integrations
+              run: |
+                  dirs=$(find integration-templates -maxdepth 1 -type d | tail -n +2)
+                  for dir in $dirs; do
+                    integration=$(basename $dir)
+
+                    mkdir -p /tmp/nango-temp/nango-integrations/$integration
+
+                    cp $dir/nango.yaml /tmp/nango-temp/nango-integrations/
+                    cp -r $dir/* /tmp/nango-temp/nango-integrations/$integration/
+
+                    cd /tmp/nango-temp/nango-integrations
```

Comment:
This is the Nth time we have done that, just because we enforce `nango-integrations` to be the folder, without too many reasons. Maybe at some point, we should just check if there is nango.yaml and that's good